resources/web/template.html

```diff
@@ -161,1115 +101,314 @@
     <!-- new ga for staging server -->
 
     <!-- Elastic APM -->
-<script src="/static/js/elastic-apm-js-base.umd.min.2.2.0.js"></script>
-
-
-<script type="text/javascript">
-  var active;
-  
-  active = Math.random() < 1 //0.9 
-  
-  elasticApm.init({
-    active: active,
-    serviceName: "elastic-co-frontend",
-    serverUrl: "https://elastic-co-apm-server.app.elstc.co"
-  })
-
-  var pageLoadName
-  var pathname = window.location.pathname.replace(/\/es|\/pt|\/jp|\/pt|\/fr|\/kr|\/cn|\/de/g, '')
-  var parts = pathname.split('/');
-
-  if (parts.length && parts[1] === 'blog') {
-    if (parts.length === 2) {
-      pageLoadName =  '/blog';
-    }
-    else if (parts[2] === 'category') {
-      pageLoadName =  '/blog/category';
-    }
-    else if (parts[2] === 'archive') {
-      pageLoadName =  '/blog/archive';
-    }
-    else {
-      pageLoadName =  '/blog/detail';
-    }
-  }
-  else if (parts.length && parts[1] === 'elasticon') {
-    if (parts.length === 2) {
-      pageLoadName =  '/elasticon';
-    }
-    else if (parts.length === 5 && (parts[2] === 'conf' || parts[2] === 'tour')) {
-      pageLoadName =  '/elasticon/overview';
-    }
-    else if (parts.length > 5 && (parts[2] === 'conf' || parts[2] === 'tour')) {
-      pageLoadName =  '/elasticon/video';
-    }
-  }
-  else if (parts.length && parts[1] === 'guide') {
-    pageLoadName =  '/guide';
-  }
-  else if (parts.length === 2 && parts[1] === 'videos') {
-    pageLoadName =  '/videos';
-  }
-  else if (parts.length > 2 && parts[1] === 'videos') {
-    pageLoadName =  '/videos/detail';
-  }
-  else if (parts.length > 2 && parts[1] === 'webinars') {
-    pageLoadName =  '/webinars/detail';
-  }
-  else {
-    pageLoadName =  window.location.pathname;
-  }
-
-  console.log('apm pageLoadName:', pageLoadName, parts)
-  elasticApm.setTags({env:"production"});
-  elasticApm.setInitialPageLoadName(pageLoadName);
-</script>
-
-<!-- Elastic APM -->
-
+    <!--
```

Comment:
Do we mean to have this commented out?

Reply:
Could we just remove it?

Reply:
I'm noticing an error when I enable APM, but we can launch without and uncomment latter in the week when it is working.